server/sendChatMessage/index.ts

```diff
@@ -7,6 +7,59 @@ import authenticate from '../src/authenticate'
 import { getUserIdForUsername } from '../src/user'
 import { MAX_MESSAGE_LENGTH } from '../src/config'
 
+const danceList = [

```

Comment:
It would probably be nice to extract these lists out into an external TS file that's imported, this is pretty noisy for a file that's mostly business logic about message sending.

Reply:
Yeah - I tried putting the whole 'dance' function into an external TS file and exporting the function, but was running into 500 server errors. I was trying to base it off shout.ts but I had some confusion around whether or it it's awaitable and how to return properly, since unlike shout or whisper it isn't doing any other operations. If you could give me tips on how to move the whole function into an external file I'd love to do that - otherwise you're right I could at least just export the arrays.